crates/freya/src/lib.rs

```diff
@@ -8,15 +8,16 @@
 //!
 //! Powered by [🧬 Dioxus](https://dioxuslabs.com) and [🎨 Skia](https://skia.org/).
 //!
-//! - [`Elements`](freya_elements::elements#structs)
-//! - [`Events`](freya_elements::elements#functions)
-//! - [`Components`](freya_components)
-//! - [`Hooks`](freya_hooks)
-//! - [`Theming`](self::_docs::theming)
-//! - [`Hot reload`](self::_docs::hot_reload)
-//! - [`Testing`](self::_docs::testing)
-//! - [`Animating`](self::_docs::animating)
-//! - [`Devtools`](self::_docs::devtools)
+//! - [Elements](freya_elements::elements#structs)
+//! - [Events](freya_elements::elements#functions)
+//! - [Elements and events guides](freya_elements::_docs)
```

Comment:
> //! - [Elements and events guides](freya_elements::_docs)

This working seems odd to me. There is no event mention of events in those docs, and leaving `Elements guides` also sounds weird to me, isn't there maybe a better way to put this?

Reply:
I used "elements and events" because I assume it might have some event docs in the future. It can be "Elements guides", but if we forget to change it once some event "guide" is added there, people might just assume there's no additional event docs other than the API.

As for the ".. guides" name, I don't really see the problem. That's what they're referred to on the website, so I just used that.

Reply:
Well, what triggers me off is that there is elements, events, and then elements & events again, reads weird to me

Reply:
I changed "Elements" and "Events" to "Elements API reference", "Events API reference". Should be more clear.

Reply:
that's weird considering all the other things from the list are also API references xD

Reply:
The other things on the list don't have "guides" though.